content/applications/general/companies/digest_emails.rst

```diff
@@ -1,83 +1,209 @@
 
-- which KPIs are shared in the Digest
-- how often Digest Emails are sent
-- who in your organization receives Digest Emails
-- creating custom Digest Email templates
-- adding additional KPIs (Studio required)
+- Which :abbr:`KPIs (key performance indicators)` are shared in the digest emails
+- How often digest emails are sent
+- Who in the organization receives digest emails
+- Creating custom digest email templates
+- Adding additional :abbr:`KPIs (key performance indicators)` (*Studio* app required)
 
 .. note::
    By default, Digest Email is *enabled*, and *Your Odoo Periodic Digest* serves as the primary
-   template, which includes all KPI measurements across your Odoo database and is sent daily to
-   administrators.
+   template, which includes all :abbr:`KPI (key performance indicator)` measurements across the Odoo
+   database and is sent daily to administrators.
+
+.. warning::
+   When creating duplicates of databases that have sending capabilities (not testing-mode) the
+   digest emails will continue to send from the duplicate database unless deactivated. To deactivate
+   the duplicate database, navigate to :menuselection:`Settings --> General Settings -->
+   Statistics`. Then deactivate the :guilabel:`Digest Emails` feature by un-ticking the checkbox and
+   click on :guilabel:`Save`.
 
 .. _digest-emails/customize-digest:
 
 Customize *Your Odoo Periodic Digest*
 =====================================
 
 To customize the default Digest Email (*Your Odoo Periodic Digest*), go to :menuselection:`Settings
---> General Settings --> Statistics --> Digest Email`, select *Your Odoo Periodic Digest* and click
-on the *external link* next to the dropdown selection.
+app --> General Settings --> Statistics --> Digest Email`. Select *Your Odoo Periodic Digest* and
+click on the :guilabel:`external link` icon next to the drop-down menu selection.
 
 A popup window appears and presents a variety of editable settings, which include:
 
-- **Digest Title** - what you want your Digest Email to be called
-- **Periodicity** - control the regimen in how often Digest Emails are sent
-- **KPIs** - check/uncheck each calculated KPI that appears in Digest Emails
-- **Recipients** - add/remove users who receive your Digest Emails
-- **Custom** - add your own KPIs (Studio required)
+- :guilabel:`Digest Name`: the name of the digest email.
+- :guilabel:`Periodicity`: control how often digest emails are sent.
+- :guilabel:`Next Send Date`: the date on which the digest email will be sent again.
+- :guilabel:`KPIs` tab: check/uncheck each calculated :abbr:`KPI (key performance indicator)` that
+  appears in digest emails.
+- :guilabel:`Recipients` tab: add/remove users who receive the digest emails.
+
+.. note::
+   The :abbr:`KPIs (key performance indicators)` can be customized using Odoo *Studio*. Additional
+   costs to the database subscription will be incurred should *Studio* need to be installed.
```

Comment:
```suggestion
   costs to the database subscription are incurred, should *Studio* need to be installed.
```
do you think linking to a Studio doc about how to customize KPIs could be useful here (if that even exists haha)

Reply:
UPDATE: I see you end this PR with a section specifically covering this topic. Great! Maybe use a ref anchor here, so curious readers can jump right to that part, if they're interested in learning more about that